### PR TITLE
Pod Details

### PR DESCRIPTION
**Pull request by adfost**
Screen Shot 2021-06-04 at 4 33 31 PM

### CODE REVIEW

**Review by dvaldivia — CHANGES_REQUESTED** [review]
Put all the content inside a <Paper> container, you can copy the styling of `Trace.tsx` look at it in action https://play.min.io:9443/trace

File: logs.log [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Log file created at: 2021/06/02 16:17:26
```
Comment (dvaldivia) [inline logs.log:1]: remove this file

**Review by dvaldivia — DISMISSED** [review]

File: portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Affinity.tsx [hunk 2]
```diff
@@ -125,7 +125,7 @@ const Affinity = ({
           setKeyValueMap({});
         });
     }
-  }, [setModalErrorSnackMessage, loading]);
+  }, []);
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Affinity.tsx:128]: These variables should not be removed as they are required for the effect to work correctly (We just need to run this effect everytime loading changes)

File: portal-ui/src/screens/Console/Tenants/TenantDetails/PodDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+  };
+
+  const renderLines = logLines.map((m, i) => {
+    return renderLog(m, i);
+  });
+
+  useEffect(() => {
+    loadInfo();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/PodDetails.tsx:239]: Please paste the content of the loadInfo() function inside the effect and remove the `// eslint-disable-next-line react-hooks/exhaustive-deps` comment. After that you just need to add the dependencies to the dependences array.

We need to fix this across the application but will be managed in other ticket

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Affinity.tsx [hunk 4]
```diff
@@ -24,12 +24,13 @@ import {
   modalBasic,
   wizardCommon,
 } from "../../../Common/FormComponents/common/styleLibrary";
-import { Grid, IconButton } from "@material-ui/core";
+import { Button, Grid, IconButton } from "@material-ui/core";
```
Comment (dvaldivia) [inline portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Affinity.tsx:27]: I think these were the changes you did by accident? Just download the latest `Affinity.tsx` from master, and replace it in your PR

**Review by dvaldivia — CHANGES_REQUESTED** [review]

**Review by dvaldivia — APPROVED** [review]

**Review by dvaldivia — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]

**Pull request merged**